### PR TITLE
level aa contrast inbox and deleted page

### PR DESCRIPTION
**Pull request by ATBull81**
# PR Checklist

Please check if your PR fulfills the following requirements
- [x] The title follows our [guidelines](https://github.com/ideacrew/enroll/blob/trunk/CONTRIBUTING.md#commit
)
- [x] Tests for the changes have been added (for bugfixes / features)

# PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature (requires Feature flag)
- [ ] Data fix or migration (inert code, no impact until run)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Dependency updates (e.g., add a new gem or update version)

# What is the ticket # detailing the issue?

Ticket: https://www.pivotaltracker.com/story/show/186499310 && https://www.pivotaltracker.com/story/show/186499309

# A brief description of the changes

Current behavior No cucumbers, no navigation through messages or mailboxes, strange setup for inbox/delete/sent messages

New behavior: Cucumber test added, functionality improved

# Feature Flag

For all new feature development, a feature flag is required to control the exposure of the feature to our end users. A feature flag needs a corresponding environment variable that is used to initialize the state of the flag. Please share the name of the environment variable below that would enable/disable the feature and which client(s) it applies to.

Variable name: CONTRAST_LEVEL_AA_IS_ENABLED

- [ ] DC
- [x] ME

# Additional Context
Include any additional context that may be relevant to the peer review process.

Screenshot 2024-01-12 at 3 32 52 PM
Screenshot 2024-01-12 at 3 32 58 PM
Screenshot 2024-01-12 at 3 33 08 PM
Screenshot 2024-01-12 at 3 34 24 PM
Screenshot 2024-01-12 at 3 35 59 PM
Screenshot 2024-01-12 at 4 09 19 PM

# AppScan CodeSweep Failure
In the event of a failed check on the AppScan CodeSweep step of our GitHub Actions workflow, please review the False Positive protocol outlined here: appscan_codesweep/CODESWEEP_FALSE_POSITIVES_README.MD

Add all required notes to this section if the failure is a suspected false positive.

### CODE REVIEW

File: app/views/insured/families/_inbox.html.erb [hunk 1]
```diff
@@ -2,39 +2,32 @@
...
+          <span>&nbsp;|&nbsp;</span>
+          <a href="<%= inbox_insured_families_path(@provider, folder: 'Sent') %>">
+            <% if @folder == 'Sent' %>
+              <strong><%= l10n("sent") %></strong>
+            <% else %>
+              <%= l10n("sent") %>
+            <% end %>
+          </a>
         <% end %>
+        <span>&nbsp;|&nbsp;</span>
```
Comment (scaustin34) [inline app/views/insured/families/_inbox.html.erb:23]: any chance we could get rid of the nbsp's?

Comment (ATBull81) [reply app/views/insured/families/_inbox.html.erb]: https://github.com/ideacrew/enroll/pull/3479/commits/f4a4d2d02791627c19d5224d34def741accbaf94

**Review by mdkaraman — APPROVED** [review]
Nice work!

**Review by scaustin34 — APPROVED** [review]
LGTM!

**Pull request merged**